apps/frontend/src/views/Login.vue

```diff
@@ -70,6 +70,10 @@ export default class Login extends Vue {
   checkLoggedIn() {
     if (ServerModule.token) {
       this.$router.push('/');
+    } else {
+      if(Vue.$cookies.get('loadEvaluation') !== null) {
+        SnackbarModule.notify("Please login to view this evaluation.")
+      }
```

Comment:
Is this leftover from the Backend changes?

Reply:
Yeah it was, removed now